people/views.py

```diff
@@ -80,6 +80,9 @@ def post(self, request, *args, **kwargs):
             # in case of failure of above try statement, all Person objects are retrieved
             queryset = Person.objects.all().order_by('id')
 
+        if phoneNumberExists in ["true","t","yes","y"]:
+            queryset = queryset.filter(phone_no__isnull=False).exclude(phone_no__in=[''])
```

Comment:
logic incorrect as it would provide numbers like 9, 0, 1 etc. Add a more generalised logic for numbers.